sources/packages/backend/apps/api/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -162,10 +163,16 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
       );
     }
 
-    await this.startAssessmentQueue.add({
-      workflowName: assessment.application.data.workflowName,
-      assessmentId: assessment.id,
-    });
+    await this.startAssessmentQueue.add(
```

Comment:
The idea of creating the object was to provide it as below. Did you find any issues following the below?
```
await this.startAssessmentQueue.add(
      {
        workflowName: assessment.application.data.workflowName,
        assessmentId: assessment.id,
      },
      QUEUE_RETRY_DEFAULT_CONFIG
    );
```